perfmetrics/scripts/hns_rename_folders_metrics/generate_folders_and_files.py

```diff
@@ -230,6 +230,16 @@ def _check_if_dir_structure_exists(directory_structure) -> bool:
   return True
 
 
+def _delete_existing_folders_and_files_in_gcs_bucket(gcs_bucket)->(int):
+  try:
+    subprocess.check_output(
```

Comment:
Add comment on deleting data on empty bucket exit the script.

Reply:
Added as TODO.